UKHomeOffice/sgmr-service
# SPCR-197 Reolve ansi-regex vulnerabilities

## Description
Inefficient Regular Expression Complexity in chalk/ansi-regex - https://github.com/advisories/GHSA-93q8-gq69-wqmw

We had two packages with dependencies using an outdated version of ansi-regex

- Update `html-webpack-plugin` to latest version
- Remove `cypress-get-table` as does not support updated versions of ansi-regex & we are reviewing Cypress config in general. Not sure what was using it as cypress tests in terminal, runner, and mochawesome reports all work as expected without it.

## Developer Checklist
- remove `package-lock` and `npm modules`
- run `npm i`
- run `npm start`
- > app should load as normal
- run `npm test`
- > tests should pass
- run `npm run cypress:runner -- --env configFile=dev`
- select `user-registration-spec`
- > tests should pass (note other tests will still fail)

\* Required






Ensure you delete the branch once the Pull Request is merged.
